### PR TITLE
POC-397: HEI summaries configurations

### PR DESCRIPTION
**Pull request by henrykorir**

### CODE REVIEW

File: src/app/patient-dashboard/hiv/hiv-summary/hiv-summary-latest.component.ts [hunk 1]
```diff
@@ -159,10 +165,11 @@ export class HivSummaryLatestComponent implements OnInit, OnDestroy {
 
   public loadHivSummary(patientUuid) {
     const summarySub = this.hivSummaryService
-      .getHivSummary(patientUuid, 0, 1, false)
+      .getHivSummary(patientUuid, 0, 1, false, this.isHEIActive)
       .subscribe(
         (data) => {
           if (data) {
+            console.log('latest-data: ', data);
```
Comment (drizzentic) [inline src/app/patient-dashboard/hiv/hiv-summary/hiv-summary-latest.component.ts]:
```suggestion
```

**Review by Alfred-Mutai — DISMISSED** [review]
LGTM

File: src/app/etl-api/hiv-patient-clinical-summary-resource.service.ts [hunk 2]
```diff
@@ -19,7 +22,8 @@ export class HivPatientClinicalSummaryResourceService {
 
     const params: HttpParams = new HttpParams()
       .set('startIndex', (0 as any) as string)
-      .set('limit', (20 as any) as string);
+      .set('limit', (20 as any) as string)
+      .set('isHEIActive', (isHEIActive as any) as string);
```
Comment (drizzentic) [inline src/app/etl-api/hiv-patient-clinical-summary-resource.service.ts]: const params: HttpParams = new HttpParams()
  .set('startIndex', '0')
  .set('limit', '20')
  .set('isHEIActive', isHEIActive?.toString() || '');

File: src/app/etl-api/hiv-patient-clinical-summary-resource.service.ts [hunk 3]
```diff
@@ -2,15 +2,18 @@ import { Injectable } from '@angular/core';
 import { AppSettingsService } from '../app-settings/app-settings.service';
 import { Observable } from 'rxjs';
 import { HttpClient, HttpParams } from '@angular/common/http';
-
+import * as Moment from 'moment';
```
Comment (drizzentic) [inline src/app/etl-api/hiv-patient-clinical-summary-resource.service.ts]: where is this being used?

File: src/app/patient-dashboard/hiv/hiv-summary/hiv-summary.service.ts [hunk 4]
```diff
@@ -16,16 +16,17 @@ export class HivSummaryService {
...
   ): Observable<any> {
     const hivSummary: BehaviorSubject<any> = new BehaviorSubject(null);
-
     this.hivSummaryResourceService
       .getHivSummary(
         patientUuid,
         startIndex,
-        this.limit,
-        includeNonClinicalEncounter
+        limit,
```
Comment (drizzentic) [inline src/app/patient-dashboard/hiv/hiv-summary/hiv-summary.service.ts]: why did we change this?

**Review by drizzentic — CHANGES_REQUESTED** [review]

**Review by drizzentic — APPROVED** [review]

**Pull request merged**